src/App.tsx

```diff
@@ -21,15 +21,15 @@ function App() {
       <Header />
       <main>
         <Routes>
-          <Route path="/" element={<HomePage />}></Route>
+          <Route path="homepage" element={<HomePage />}></Route>
```

Comment:
usually, "/" means exactly the home page)
so probably, it's better to revert this change